tests/test_computeNextBump.py

```diff
@@ -0,0 +1,259 @@
+import DotBot
+import Floorplan
+import pytest
+import random
+
+# ============================ fixtures ==============================
+
+EXPECTEDINOUT = [
+
+    # diagonally towards obstacle (left to right)
+    {
+        'in': {
+            'currentX': 0.00,
```

Comment:
indent the values so the dots are aligned. A general good rule to follow as it allows you to easily compare values